instrumented/benchmark/gradle.properties

```diff
@@ -1 +1 @@
-android.enableAdditionalTestOutput=true
\ No newline at end of file
+android.enableAdditionalTestOutput = true
```

Comment:
Why did you change this ? Not sure it'll keep working everywhere :/

Reply:
I didn't...maybe it was ktlint.....I will put it back ;)